src/Raven.Client/Documents/Commands/DeleteDocumentCommand.cs

```diff
@@ -30,4 +30,4 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
             return request;
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Let's avoid including files that haven't really changed.

Reply:
done